src/main.js

```diff
@@ -1,19 +1,50 @@
 // Create variables targetting the relevant DOM elements here 👇
-
+var coverPic = document.querySelector('.cover-image')
+var coverTitle = document.querySelector('.cover-title')
+var coverTagLine1 = document.querySelector('.tagline-1')
+var coverTagLine2 = document.querySelector('.tagline-2')
+var priceTag = document.querySelector('.price-tag')
+var randomBtn = document.querySelector('.random-cover-button')
 
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
-var currentCover;
+
 
 // Add your event listeners here 👇
 
 
+randomBtn.addEventListener('click', randomizeCover);
+
+
 // Create your event handlers and other functions here 👇
+function randomizeCover() {
+  var randomNumC = getRandomIndex(covers);
+  var randomNumT = getRandomIndex(titles);
+  var randomNumD = getRandomIndex(descriptors);
+  var randomNumD2 = getRandomIndex(descriptors);
```

Comment:
I think we can refactor this "randomNumX" part of the code so that it's more concise. Check out this syntax for inspiration:
`var bookTitle = descriptors[getRandomIndex(descriptors)]`
Can you figure out what that line would do?
These lessons or your notes on these topics should be able to help:
https://frontend.turing.edu/lessons/module-1/js-intro-to-functions.html
https://frontend.turing.edu/lessons/module-1/js-dot-bracket-notation.html